### PR TITLE
Match atk5D_getmoneyreward

### PR DESCRIPTION
**Pull request by PokeCodec**
This one was so annoying, because I would never have thought to put a pointer for all 4 types of structs in the beginning, but then I remembered this is GF, so. It took a ton of permutations.

I am tired.

### CODE REVIEW

File: src/battle_script_commands.c [hunk 1]
```diff
@@ -5155,66 +5155,69 @@ static void atk5C_hitanimation(void)
...
+    const struct TrainerMonNoItemDefaultMoves *party1;
+
+    const struct TrainerMonNoItemCustomMoves *party2;
+
+    const struct TrainerMonItemDefaultMoves *party3;
+
+    const struct TrainerMonItemCustomMoves *party4;
+
 
     // The whole function is using wrong registers.
```
Comment (ketsuban) [inline src/battle_script_commands.c:5173]: If it's matched now, do you still need this comment?

Comment (PokeCodec) [reply src/battle_script_commands.c]: No

**Comment by PokeCodec** [comment]
Fixed

**Comment by PokeCodec** [comment]
Done!

**Pull request closed without merging**